### PR TITLE
0.10.3 plus structure

### PR DESCRIPTION
**Pull request by josmo**
@jmccann Let me know if there's anything different you want. I tried to put in a few of the patterns from the drone-plugins org and update to 0.10.3

### CODE REVIEW

**Comment by josmo** [comment]
Don't merge this yet.  There's a couple issues with the dockerfile. I'll fix and update

**Comment by josmo** [comment]
I just commented out from scratch till the deps get figured out. It should be good to go

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # drone-terraform
 
 [![Build Status](http://beta.drone.io/api/badges/jmccann/drone-terraform/status.svg)](http://beta.drone.io/jmccann/drone-terraform)
+[![Join the chat at https://gitter.im/drone/drone](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/drone/drone)
```
Comment (jmccann) [inline README.md]: This is going away, so probably don't want a link to it

File: README.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
+[![Go Doc](https://godoc.org/github.com/jmccann/drone-terraform?status.svg)](http://godoc.org/github.com/jmccann/drone-terraform)
```
Comment (jmccann) [inline README.md]: There didn't seem much value provided by godoc so I think we can leave that out

File: README.md [hunk 3]
```diff
@@ -19,7 +23,7 @@ go test
 Build the docker image with the following commands:
 
 ```
-CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -tags netgo
+CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -tags netgo -o release/linux/amd64/drone-terraform
```
Comment (jmccann) [inline README.md]: Don't think we need to change this

**Comment by jmccann** [comment]
Also, I've merged/committed a bit since this PR ... so you may want to rebase from `master`

**Comment by jmccann** [comment]
Closing for now as it seems to have gone stale and per my comments from earlier.

**Pull request closed without merging**